### PR TITLE
Refactored Archi to use layouts in companies and talents folders 

### PR DESCRIPTION
**Pull request by brunolune**
Only modified archi, didn't  include metadata yet, and not modified NavBar (maybe no need to ...)

### CODE REVIEW

File: app/companies/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import "../globals.css";
+
+import "@rainbow-me/rainbowkit/styles.css";
+import { Rainbowkit } from "../components/rainbowkit";
```
Comment (n1c01a5) [inline app/companies/layout.tsx:6]: Add breakline after l5.

File: app/companies/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
```
Comment (n1c01a5) [inline app/companies/layout.tsx:7]: Remove this line

File: app/companies/layout.tsx [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+}) {
+  return (
+    <html lang="en">
+      <body>
+        <Rainbowkit>
+          <NavBar />
+        </Rainbowkit>
+        {children}
+        <footer className="mx-5 text-sm">
+          <a
```
Comment (n1c01a5) [inline app/companies/layout.tsx:23]: Just remove this bad link.

File: app/talents/layout.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+"use client";
```
Comment (n1c01a5) [inline app/talents/layout.tsx:1]: Same here.

File: app/companies/layout.tsx [hunk 5]
```diff
@@ -0,0 +1,34 @@
+"use client";
+
+import "../globals.css";
+
+import "@rainbow-me/rainbowkit/styles.css";
+import { Rainbowkit } from "../components/rainbowkit";
+
+import { NavBar } from "../components/nav-bar";
+
+export default function CompaniesLayout({
```
Comment (n1c01a5) [inline app/companies/layout.tsx:10]: Just use `Layout`.

File: app/talents/layout.tsx [hunk 6]
```diff
@@ -0,0 +1,34 @@
...
+
+import "../globals.css";
+
+import "@rainbow-me/rainbowkit/styles.css";
+import { Rainbowkit } from "../components/rainbowkit";
+
+import { NavBar } from "../components/nav-bar";
+
+export default function TalentsLayout({
```
Comment (n1c01a5) [inline app/talents/layout.tsx:10]: Just use `Layout`.

**Review by n1c01a5 — CHANGES_REQUESTED** [review]

File: app/companies/layout.tsx (hunk 1 above)
Comment (brunolune) [reply app/companies/layout.tsx]: done

File: app/companies/layout.tsx (hunk 2 above)
Comment (brunolune) [reply app/companies/layout.tsx:6]: done

File: app/companies/layout.tsx (hunk 3 above)
Comment (brunolune) [reply app/companies/layout.tsx]: done

File: app/talents/layout.tsx (hunk 4 above)
Comment (brunolune) [reply app/talents/layout.tsx:1]: done

File: app/companies/layout.tsx (hunk 5 above)
Comment (brunolune) [reply app/companies/layout.tsx]: done

File: app/talents/layout.tsx (hunk 6 above)
Comment (brunolune) [reply app/talents/layout.tsx]: done

**Review by n1c01a5 — APPROVED** [review]

**Pull request merged**